Tasks/CopyFilesOverSSHV0/sshhelper.ts

```diff
@@ -138,13 +140,68 @@ export class SshHelper {
         return defer.promise;
     }
 
+    /**
+     * Returns true if the remote path exists and is a directory
+     * @param path
+     * @returns {Promise<boolean>}
+     */
+    public async checkRemotePathIsADirectory(path: string): Promise<boolean> {
```

Comment:
I didn't end up using this function, but decided that it was simple enough that deleting the code was a wash.  Happy to take it out in the interest of minimizing non-exercised code if needed.